bin/run.sh

```diff
@@ -31,20 +31,38 @@ mkdir -p "${output_dir}"
 
 echo "${slug}: testing..."
 
+TEST="${input_dir}/${slug}.spec.wren"
+
 # Run the tests for the provided implementation file and redirect stdout and
 # stderr to capture it
-# TODO: Replace 'RUN_TESTS_COMMAND' with the command to run the tests
-test_output=$(RUN_TESTS_COMMAND 2>&1)
+ln -sf ../../../vendor ${input_dir}/vendor
+rm -f $results_file
+test_output=$(wren_cli $TEST $results_file 2>&1)
+
+status=$?
+
+# cleanup
+rm ${input_dir}/vendor
+# echo "$test_output"
+
+# if we have a results file that was generated by the test runner
+# then use that, augment with stack trace if necessary
+if [ -f $results_file ]; then
+    cat ${results_file} | jq -M > .tmp
```

Comment:
What does `jq -M` do? I don't see any query.

Reply:
monochrome, but looks like it's not necessary.